apps/src/music/views/timeline.module.scss

```diff
@@ -90,6 +90,16 @@
           border-color: violet;
```

Comment:
We can probably remove the `&Pink` and `&Orange` classes if they're not used anymore?

Reply:
I thought it might be nice to maintain a library of known colour schemes, but fairly neutral on that :)

Reply:
Okay, removing for now.